api/src/paths/project/{projectId}/survey/{surveyId}/observation/index.ts

```diff
@@ -0,0 +1,327 @@
+import { RequestHandler } from 'express';
+import { Operation } from 'express-openapi';
+import { authorizeRequestHandler } from '../../../../../../request-handlers/security/authorization';
+import { getLogger } from '../../../../../../utils/logger';
+import { PROJECT_PERMISSION, SYSTEM_ROLE } from '../../../../../../constants/roles';
+import { getDBConnection } from '../../../../../../database/db';
+import { ObservationService } from '../../../../../../services/observation-service';
+import { InsertObservation, UpdateObservation } from '../../../../../../repositories/observation-repository';
+
+const defaultLog = getLogger('/api/project/{projectId}/survey/{surveyId}/observation/get');
```

Comment:
is the default log get? maybe just leave at `/api/project/{projectId}/survey/{surveyId}/observation/`

Reply:
Good catch